libs/ledger-live-common/src/exchange/swap/hooks/useSwapTransaction.ts

```diff
@@ -1,23 +1,22 @@
 import { AmountRequired, NotEnoughGas, NotEnoughGasSwap } from "@ledgerhq/errors";
 import { useMemo } from "react";
 import {
-  ExchangeRate,
   SwapSelectorStateType,
   OnNoRatesCallback,
   SwapTransactionType,
-  AvailableProviderV3,
-  OnBeforeTransaction,
+  OnBeforeFetchRates,
```

Comment:
Can we remove "onBeforeFetchRates" & "onBeforeFetch"? Do you have something in mind? (Not sure if we really need it...)

Reply:
Will check to see if it's used somewhere, but yes, happy to remove if not used.